.circleci/config.yml

```diff
@@ -61,6 +62,20 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - test-render-linux-firefox:
+          requires:
+            - prepare
+          filters:
+            tags:
+              only: /.*/
+      - test-render-macos-chrome:
+          filters:
+            tags:
+              only: /.*/
+      - test-render-windows-chrome:
```

Comment:
It looks like the artifact showing the render test results are missing from the builds (refer [here](https://app.circleci.com/pipelines/github/mapbox/mapbox-gl-js/11968/workflows/c86bedcd-5b1d-4afb-ae2d-65d706a14c64/jobs/159337/artifacts) and [here](https://app.circleci.com/pipelines/github/mapbox/mapbox-gl-js/11968/workflows/c86bedcd-5b1d-4afb-ae2d-65d706a14c64/jobs/159349/artifacts)). Is there something preventing the builds to store the build artifact?

Reply:
I'm not able to reproduce that on more recent runs. Not sure what happened there, maybe hitting some sort of flakiness related to storing artifacts on virtual machines?

Reply:
Not sure, maybe the build got stale and the artifacts were not available anymore.